OFL.txt

```diff
@@ -1,8 +1,8 @@
-Copyright (c) 2023 Intel Corp. with Reserved Font Name "Intel One Mono", "IntelOne Mono"
+Copyright 2023 The Intel One Mono Project Authors (https://github.com/intel/intel-one-mono), with Reserved Font Name 'Intel One Mono', 'IntelOne Mono'
```

Comment:
Remove reserved font name?

Reply:
Alas no, they gave us RFN permission tho

Reply:
Agreed, we are fine with Reserved Font Name 'Intel.